lang/main.json

```diff
@@ -385,8 +385,10 @@
         "externalInstallationMsg": "You need to install our desktop sharing extension.",
         "inlineInstallationMsg": "You need to install our desktop sharing extension.",
         "inlineInstallExtension": "Install now",
+        "kickParticipantDialog": "Are you sure you want to kick this participant?",
         "muteParticipantTitle": "Mute this member?",
         "muteParticipantBody": "You won't be able to unmute them, but they can unmute themselves at any time.",
+        "muteParticipantDialog": "Are you sure you want to mute this participant? You won't be able to unmute them, but they can unmute themselves at any time.",
```

Comment:
This is a combination of the above two strings for convenience and/or better internationalization?

Reply:
No, this is slightly different from web, because we don't have a dialog title on mobile now (see the new dialogs), and the web message only made sense together with the dialog title.